### PR TITLE
Fix error reading pagination when no reportbacks exist for campaign.

### PR DESCRIPTION
**Pull request by DFurnes**
#### Changes
Phoenix Ashes [doesn't return a `meta.pagination` object](https://cloud.githubusercontent.com/assets/583202/25008075/9395d234-2030-11e7-931a-f60d7433920b.png) if there aren't any results for a query, which means we were throwing an exception when trying to read that in the response. This pull request uses Lodash's [`get`](https://lodash.com/docs/#get) helper to gracefully fail if that property isn't defined.

📟

### CODE REVIEW

**Comment by DFurnes** [comment]
[🐛 As a Phoenix-next user, I want to be able to still click "view more" even if there aren't reportbacks loading 🐛](https://trello.com/c/f72nHZcI/517-as-a-phoenix-next-user-i-want-to-be-able-to-still-click-view-more-even-if-there-aren-t-reportbacks-loading)

File: resources/assets/actions/reportback.js [hunk 1]
```diff
@@ -208,7 +209,10 @@ export function fetchReportbacks() {
 
     (new Phoenix).get('next/reportbacks', { campaigns: node, page }).then(json => {
       const normalizedData = normalizeReportbacksResponse(json.data);
-      dispatch(receivedReportbacks(normalizedData, json.meta.pagination.current_page, json.meta.pagination.total))
+      const currentPage =  get(json, 'meta.pagination.current_page', 1);
```
Comment (weerd) [inline resources/assets/actions/reportback.js]: 👾

File: resources/assets/actions/reportback.js [hunk 2]
```diff
@@ -208,7 +209,10 @@ export function fetchReportbacks() {
...
+      const total =  get(json, 'meta.pagination.current_page', 0);
```
Comment (weerd) [inline resources/assets/actions/reportback.js]: 👾

Comment (weerd) [inline resources/assets/actions/reportback.js]: Should this be referencing `total` and not `current_page`?

Comment (DFurnes) [inline resources/assets/actions/reportback.js]: Oh it sure should. 🙃 ☕️

**Pull request merged**